ol/documentation/node-ops/validators/validator-requirements.md

```diff
@@ -0,0 +1,45 @@
+This will hopefully be beneficial for the network where active validators will endorse contributors, people with special skills or equipments for the well-being or improvments of 0L. 
+The vouches also allow some kind of weak security against sybils. For an example of what exising validators might look for when deciding to vouch for a new validator, see [these requirements](https://hackmd.io/-Zi5b2HTT46FxHGWW5nSlw).
+
+    - Family ancestry:
+One must collect at least 4 vouches from 4 different families.
+A family is defined as accounts sharing common ancesters. 
+
+    - The command to provide to voucher ```txs vouch --address <<<YOUR_ADDRESS>>>``` 
+    
+    - The command to see your collected vouches ```cd ~/libra && cargo r -p diem-transaction-replay -- --db ~/.0L/db annotate-account <<<YOUR_ADDRESS>>>```
+
+4. Tower Height - Be one of the validators with the highest tower height among validators who have met the other five criteria in this list;
+5. Voting Power - At the time of writing, the validator set can inflate of 15% maximum. Therefore, if more validators verify the requirements, the validators with the most voting power will pass in the active set first.
+
+For example, if only three validators will be admitted at the epoch change, the 3 validators who have been onboarded, submitted 8 proofs, collected 4 vouches and have the highest tower height and greatest voting power will be admitted into the set.
+
+Additional resources to bring - 
+
+- An existing validator back into the active set, see [here](https://hackmd.io/F1gSIRn8TLubg8i3TG_gpA).
+- A new validator online, see [here](https://docs.google.com/document/d/1sAyX04cfWeUtEnWsm9ezsjQgkjE5SVstc2Z4nRtZFJM/edit).
+
+# To stay in the active set
+
+All active validators MUST keep satisfying the requirements of entry, otherwise it will drop off the active set. 
+In addition, the active validators must vote at least on 5% of the blocks during the epoch. If one validator fails to vote enough, it will drop.
+
+# Inactive validators
+
+All validators (active AND inactive) are paying a fee called '_entry fee_' for the _privilege_ of being a validator.
+The entry fee is 50% of the validator payment, for example, let's assume validator are being paid 20k coins every validator will pay 10k at the begining of the epoch.
+Therefore, inactive validators will be paying the entry fee but not receiving the payment for not being active.
+
+# Entry fee
+
+The entry fees collected are redistributed to community wallets, hence replacing the autopay mechanism **partially**. The community wallets are not receiving an equal pay from these fee. Their share is equivalent to the amount validators are giving through their autopay. The community wallets receiving the most donations will receive a biggest chunk of the entry fees as additionnal income source. 
```

Comment:
```suggestion
The entry fees collected are redistributed to community wallets, hence replacing the autopay mechanism **partially**(Autopay is voluntary and can be utilized as an extra donation on top of the entry fee to influence the index of [match burn](https://github.com/OLSF/libra/blob/151f069d41c41b8a65354cc78858003614145d10/ol/documentation/accounts/match_burn.md)). The community wallets are not receiving an equal pay from these fee. Their share is equivalent to the amount validators are giving through their autopay. The community wallets receiving the most donations will receive a biggest chunk of the entry fees as additionnal income source. 

> **Balance below entry fee:** If a validator account has a balance below the entry fee for a given epoch but meets the above requirements, the cost to enter the set will be the remaining balance in their account. If the validator is a new entrant and has ~0 balance they are allowed entry to the set if they meet the other requirements. 

```

Reply:
Should there be information explaining what happens when a validator's balance is below the entry fee?

Reply:
> (Autopay is voluntary and can be utilized as an extra donation on top of the entry fee to influence the index of [match burn]

Also maybe adding some information about Autopay and linking to the Match Burn document for context

Reply:
The Entry fee must be re-written. I was not aware of 0Ds doc at the time of writting,